src/search/search.component.spec.ts

```diff
@@ -24,14 +24,14 @@ describe("Search", () => {
 			],
 			providers: []
 		})
-		.overrideComponent(Search, {
-			remove: {
-				templateUrl: "search.component.html"
-			},
-			add: {
-				template: require("./search.component.html")
-			}
-		})
+		// .overrideComponent(Search, {
+		// 	remove: {
+		// 		templateUrl: "search.component.html"
+		// 	},
+		// 	add: {
+		// 		template: require("./search.component.html")
+		// 	}
+		// })
```

Comment:
Is this gonna be added back in eventually?

Reply:
Probably! The tests were very unhappy with that for some reason, so I'll see if I can get it working now

Reply:
Checking it out it looks like that was a workaround from when the tests didn't know how to deal with `templateUrl` ... they do now so we can safely remove that!